javascript/javascript2/week2/doubling.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title></title>
+
+    </head>
+
+    <body>
+
+
+        <script type="text/javascript">
+            let numbers = [1, 2, 3, 4];
+            let newNumbers = [];
+            let doubledNumbers = [];
+
+           
+
+            const oddNumber = number=> number % 2 !== 0;
+            const double = oddNum=> { console.log("The doubled numbers are", oddNum*2);}
+            newNumbers = numbers.filter(oddNumber);
```

Comment:
```suggestion
            const newNumbers = numbers.filter(oddNumber);
```

Then delete the newNumbers array above

Reply:
I changed the whole code to the following 3 lines. Should I delete this file on github?

`const numbers = [1, 2, 3, 4];
const oddNumbers = numbers.filter(oddNum => oddNum % 2 !== 0)
oddNumbers.forEach( num=>console.log("The doubled numbers are "+ 2*num))`

Reply:
I changed the whole code to the following 3 lines. Should I delete this file on github?

const numbers = [1, 2, 3, 4]; 
const oddNumbers = numbers.filter(oddNum => oddNum % 2 !== 0) 
oddNumbers.forEach( num=>console.log("The doubled numbers are "+ 2*num))

Reply:
No need to delete. Just change the file and the add the change, commit it and push it